Tickets/cap 824

patrickingraham opened the pull request.

Slimleashma reviewed (commented):
- In part 1 introduction: "The button located in LOVE to rapidly stop all telescope and dome motion. This button should be readily available and pressued by observers when experiencing and earthquake." 
Will it rapidly or gracefully stop? Change "pressued by" to "pressed by"

- "Upon installation of the GIS, the E-stop will also be activated after several seconds, once the motion has ceased from the controlled stop above"
Can you elaborate on the difference between E-stop and TCS stop all here? Will this require also the E-stop recovery? Link?

- Section 2.1 needs pictures that I currently don't have - need to go to dome and review steps and take pictures for procedure

- Section 2.2 needs pictures in addition to section 2.1 - to be done on next AuxTel visit

- 2.2.2 - 	"A spike to 1e-3 Torr may occur but should go down to ~7e-8 Torr or smaller after approximately 5 minutes. If this does not occur then the vacuum has been ruptured and the instrument must be warmed up immediately."
Is there a warm-up procedure? It seems from the text that LATISS will warm up by itself if things are bad or leaking. Are there additional steps to initiate a safe warm-up not described here?

- Sections 3 and 4: Should it be specified that TCS button be pressed when we feel a shake?

patrickingraham commented:
Bullet 1: Addressed (it'll gracefully stop). 
Bullet 2: added a sentence and linked to the E-stop reset procedure. Admittedly this procedure is in a pretty old place, but we'll get that sorted.
Bullets 3-4: Great! Let me know when you have the photos and I'll work them in.
Bullet 5: There is the start of a (safe) warm-up procedure here (https://tstn-006.lsst.io/#maintenance-procedures) but it needs work. It's on my to-do list. The emergency warm-up is the scary one. It's also on my to do list... but it's basically just turn everything off and pray. If you can connect the vacuum pump then do it. I can add a link if you think it'd be helpful...
Bullet 6: So this is in the big red part of the intro but I also added a sentence to the top of each section.

Slimleashma reviewed (commented):
Looks good to me, and the pictures are helpful.
